### DISCUSSION TITLE
CoPilot chat is not enabled on IntelliJ IDEs

### DISCUSSION THREAD

**Discussion by golanshy** [community/community#108015] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

CoPilot chat is not available on GoLand although I'm logged in to my GitHub account and that it is enabled on GitHub:

Suggestions matching public code - Alllowed

Copilot Chat in the IDE - enabled

GitHub Copilot in the CLI Beta - enabled

I tried logout and back in again and it is still showing as not available.
I contacted IntelliJ support and they have advised to contact GitHub:

"Could you please fill out a new ticket on https://github.com/orgs/community/discussions?
GitHub controls the access and plugin itself so there is no way to troubleshoot it somehow from JetBrains (IDE) side."

Thanks for your help.

![Screenshot 2024-02-09 at 15 32 56](https://github.com/community/community/assets/1773923/f276986f-902f-4349-b2b0-46641e512ee6)
<img width="1399" alt="Screenshot 2024-02-09 at 15 31 42" src="https://github.com/community/community/assets/1773923/c923e2b3-acb0-440f-ab28-a0cfb5f4e7c5">

**Comment by highhigher** [comment]
Could someone from github copilot team answer this?

**Comment by davecheney** [comment, marked as answer]
@golanshy @highhigher Chat in IDEA editors is a waitlisted feature. https://github.blog/changelog/2023-11-08-copilot-in-jetbrains-ides-chat-is-now-available-in-private-beta-new-welcome-guide/

**Reply by renatovalezin** [reply]
Do you know about any scheduled date for the end of the private beta end and the feature to be GA?

**Comment by thienkb1123** [comment]
I have the same issue
